gobblin-utility/src/main/java/org/apache/gobblin/broker/SharedResourcesBrokerImpl.java

```diff
@@ -260,7 +260,7 @@ public SubscopedBrokerBuilder withOverridingConfig(Config config) {
             parentScopes.add(createWrappedScope(tpe.defaultScopeInstance(), scopeMap, mainScopeType));
           } else {
             throw new IllegalArgumentException(String.format(
-                "Scope %s is an ancestor of %s, however it does not have a default id and is not provided as an acestor scope.",
+                "Scope %s is an ancestor of %s, however it does not have a default id and is not provided as an acenstor scope.",
```

Comment:
Still a typo. Recent versions of IntelliJ should catch and highlight typos - https://www.jetbrains.com/help/idea/spellchecking.html

Reply:
Will address !